src/main/resources/application.yml

```diff
@@ -91,25 +91,25 @@ temporary-certificate:
     english:
       path: "classpath:temporary-certificate-template/Explainer Document (English).pdf"
       placeholder:
-        ero-name: "eroName"
```

Comment:
The field names in the template PDFs have changed (unsurprisingly)

Reply:
It's almost as if we knew that would happen and went to the effort of externalising their names 😄